src/components/templates/EmailTemplate2.vue

```diff
@@ -0,0 +1,205 @@
+                  >
+                    <tbody>
+                      <!-- Phone number field -->
+                      <tr>
+                        <td>
+                          <span>{{ otherFields[2].value }}</span>
+                        </td>
+                      </tr>
+                      <!-- Email address & website fields -->
+                      <tr>
+                        <td>
+                          <span v-if="(otherFields[1].value || otherFields[0].value)">
+                            <a
+                              style="text-decoration: none; color: inherit;"
+                              :href="`mailto:${otherFields[1].value}`"
+                            >{{ otherFields[1].value }}</a> -
+                            <a
+                              style="text-decoration: none; color: inherit;"
+                              :href="`${formatLink(otherFields[0].value
+                              )}`"
+                            >{{ otherFields[0].value }}</a>
+                          </span>
+                        </td>
+                      </tr>
+                      <!-- Other fields -->
+                      <template v-for="(item, index) in otherFields">
+                        <tr
+                          v-if="item.value && index > 2"
+                          :key="item.name"
+                        >
+                          <td>
+                            <a
+                              v-if="item.type === 'link'"
+                              :href="formatLink(item.value)"
+                              style="text-decoration: none; color: inherit;"
+                            >{{ item.value }}</a>
+                            <a
+                              v-if="item.type === 'email'"
+                              :href="`mailto:${item.value}`"
+                              style="text-decoration: none; color: inherit;"
+                            >{{ item.value }}</a>
+                            <span v-if="item.type === 'text'">{{ item.value }}</span>
+                          </td>
+                        </tr>
+                      </template>
+                      <!-- Social icons -->
+                      <tr>
+                        <td>
+                          <table
+                            cellspacing="2"
+                            cellpadding="0"
+                            border="0"
+                            role="presentation"
+                            style=" margin-top: 5px; margin-left: -2px;"
+                          >
+                            <tbody>
+                              <tr>
+                                <td
+                                  v-for="item in socials.installed"
+                                  :key="item.name"
+                                  align="center"
+                                  style="width: 24px; height: 24px; border-radius: 3px;"
```

Comment:
I think that icon should be a 20px like templates 1 & 2

Reply:
The design I was looking at had slightly bigger icons that's why I made it 24px. I'll reverse it.

Reply:
Yes, but i think what we need little adapt the design for more consistency with exist template